test/kubemark/run-e2e-tests.sh

```diff
@@ -39,6 +39,6 @@ fi
 
 # Running locally.
 for ((i=0; i < ${#ARGS[@]}; i++)); do
-  ARGS[$i]="$(echo "${ARGS[$i]}" | sed -e 's/\[/\\\[/g' -e 's/\]/\\\]/g' )"
+  ARGS[i]="$(echo "${ARGS[$i]}" | sed -e 's/\[/\\\[/g' -e 's/\]/\\\]/g' )"
```

Comment:
I didn't know you could do this

Reply:
why inside you have to use `$i` and outside no?

Reply:
because bash.. <shrug>